### ISSUE TITLE
Understanding `server-force-leave`

### ISSUE THREAD

**Issue by Supernomad** [hashicorp/nomad#844]
So I ran into an unfortunate issue with my configuration management scripts that accidentally added new nodes to an existing nomad server cluster. 

Realizing my mistake I quickly killed the nodes and issued `nomad server-force-leave <node-name>`, which seems to have worked as the nodes are now labled as left in `nomad server-members` output. However this doesn't seem to have helped my situation, as I am left with a cluster without a leader.

What I was expecting is that `server-force-leave` would remove the specified node, and a new election would take place either right then or scheduled for some time in the near future. This doesn't seem to be the case, or am I doing something wrong?

TL;DR
What exactly is `server-force-leave` for? Also how exactly does one fix a quorum loss without nuking the cluster, or is that really the only option?

**Comment by dadgar** [comment]
`server-force-leave` is for transitioning a server in state `failed` to `left`. This is in the event that a machine fails and you want it to leave the cluster. I will be adding some documentation on how to recover the cluster for the 0.3.2 release.
